dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -73,6 +91,10 @@ dom0-rpc-qubes.r5-format-ask-allow:
         securedrop.Proxy        *           sd-app sd-proxy allow
 
         qubes.Gpg               *           @tag:sd-client sd-gpg allow
+        qubes.GpgImportKey      *           @tag:sd-client sd-gpg allow
```

Comment:
We can't test this until the other bookworm vm issues are resolved, but for parity with the old style format, we need this (https://github.com/freedomofpress/securedrop-workstation/pull/970/files#diff-c3b8900707dbf877b778a2d4222b636663d737f039290adc74585335acba24b8L88-L97)

Reply:
Bookworm kernel issues are resolved, and on my machine including this line allows for import of new source keys.